app/main.py

```diff
@@ -1 +1,57 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(
+            self, name: str,
+    ) -> None:
```

Comment:
one parameter per line, or here you can simply write your __init__ at one line, because you have only 2 params